service/src/test/java/com/sedliarov/learningtable/integration/StudentControllerIntegrationTests.java

```diff
@@ -84,119 +88,110 @@ void testGetStudents() {
   }
 
   @Test
-  void negativeTestGetStudents() {
-
-    // given
-    List<StudentDto> expectedStudents = new ArrayList<>();
-
+  void testGetStudentsIfNoUsers() {
     // when
     ResponseEntity<StudentDto[]> response = exchangeGetWithoutAuth(STUDENTS_URL, StudentDto[].class);
 
     // then
-    assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
-    assertThat(response.getBody()).containsExactlyElementsOf(expectedStudents);
+    assertSoftly(softly -> {
+      softly.assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
+      softly.assertThat(response.getBody()).containsExactlyElementsOf(Collections.emptyList());
+    });
   }
 
   @Test
   void testCreateStudent() {
-
     // given
-    StudentDto newStudent = StudentFixture.createDto();
+    StudentDto expectedStudent = StudentFixture.createDto();
 
     // when
-    ResponseEntity<StudentDto> student =
-        exchangeAddWithoutAuth(STUDENTS_URL, newStudent, StudentDto.class);
+    ResponseEntity<StudentDto> response =
+        exchangePostWithoutAuth(STUDENTS_URL, expectedStudent, StudentDto.class);
 
     // then
-    assertThat(student.getStatusCode()).isEqualTo(HttpStatus.CREATED);
-    assertThat(student.getBody().getFirstName()).isEqualTo(newStudent.getFirstName());
-    assertThat(student.getBody().getSecondName()).isEqualTo(newStudent.getSecondName());
-    assertThat(student.getBody().getNote()).isEqualTo(newStudent.getNote());
-    assertThat(student.getBody().getGroup()).isEqualTo(newStudent.getGroup());
+    expectedStudent.setStudentId(response.getBody().getStudentId());
+    assertSoftly(softly -> {
+      softly.assertThat(response.getStatusCode()).isEqualTo(HttpStatus.CREATED);
+      softly.assertThat(response.getBody()).isEqualTo(expectedStudent);
+    });
```

Comment:
```suggestion
      assertThat(response.getStatusCode()).isEqualTo(HttpStatus.CREATED);
      assertThat(response.getBody()).isEqualTo(expectedStudent);
```

Reply:
softly uses if you have more than 3 assert blocks

Reply:
Done

Reply:
Done